_includes/head.html

```diff
@@ -78,6 +78,10 @@
 ================================================== -->
 <script src="{{ site.baseurl }}/assets/js/jquery-2.1.1.min.js"></script>
 <script src="{{ site.baseurl }}/assets/js/responsiveslides.min.js"></script>
+<script src="{{ site.baseurl }}/assets/components/angular/angular.min.js"></script>
```

Comment:
I think js scripts should be loaded at the bottom of the page, unless using the async attribute.